src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/PostClaimOrchestrationHandler.java

```diff
@@ -110,8 +116,13 @@ public void citizenIssueHandler(CitizenClaimCreatedEvent event) {
                 .andThen(c -> notifyClaimantOperation.perform(c, event))
                 .apply(claim);
 
-            if (updatedClaim.getState().equals(ClaimState.CREATE)) {
+            if (Objects.equals(ClaimState.CREATE, updatedClaim.getState())) {
```

Comment:
This scans better

```suggestion
            if (updatedClaim.getState() == ClaimState.CREATE) {
```

Reply:
We have an unhealthy relationship with enums and equality, but I agree! _Vive la revolution_!